frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts

```diff
@@ -284,14 +292,16 @@ export class CombinedDiskFactory {
         volumeWrapper.getType() === VolumeType.PERSISTENT_VOLUME_CLAIM
```

Comment:
we can try to lookup and assign the pvc here when the volumeType is `VolumeType.DATA_VOLUME`